app/pages/sections/call-off-ordering-party/controller.js

```diff
@@ -3,14 +3,29 @@ import { getContext } from './contextCreator';
 import { getEndpoint } from '../../../endpoints';
 import { logger } from '../../../logger';
 
-export const getCallOffOrderingPartyContext = async ({ orderId, accessToken }) => {
-  const callOffOrgDataEndpoint = getEndpoint({ endpointLocator: 'getCallOffOrderingParty', options: { orderId } });
-  const callOffOrgData = await getData({ endpoint: callOffOrgDataEndpoint, accessToken, logger });
+export const getCallOffOrderingPartyContext = async ({ orderId, orgId, accessToken }) => {
+  try {
+    const callOffOrgDataEndpoint = getEndpoint({ endpointLocator: 'getCallOffOrderingParty', options: { orderId } });
+    const callOffOrgData = await getData({ endpoint: callOffOrgDataEndpoint, accessToken, logger });
 
-  if (callOffOrgData) {
-    logger.info(`Call off ordering party found in ORDAPI for ${orderId}: ${callOffOrgData}`);
-    return getContext({ orderId, data: callOffOrgData });
+    if (callOffOrgData) {
+      logger.info(`Call off ordering party found in ORDAPI for ${orderId}: ${callOffOrgData}`);
+      return getContext({ orderId, data: callOffOrgData });
+    }
+    logger.error(`No call off ordering party data returned from ORDAPI for ${orderId}`);
+    throw new Error();
```

Comment:
why throw and then catch the Error we can just proceed with the rest of the logic as is?

Reply:
see above

Reply:
I don't think we need to throw a new error then. The 404 will throw the error and the catch then should work.

Reply:
yep removed

Reply:
had to put throw at end to throw the inner errors out